### PR TITLE
6528 - Listview Translation

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This pull request will add an additional translation for records selected in listview.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/6528

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/listview/example-multiselect.html
- Select an item
- It should be `Record(s) Selected shown`

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by deep7102 — APPROVED** [review]

File: src/components/locale/cultures/en-US.js [hunk 1]
```diff
@@ -449,6 +449,7 @@ Soho.Locale.addCulture('en-US', {
     Range: { id: 'Range', value: 'Range', comment: 'Range for tooltip' },
     RecordsPerPage: { id: 'RecordsPerPage', value: '{0} Records per page', comment: 'Shows an actual number of pages in the pager' },
     RecordsPerPageNoToken: { id: 'RecordsPerPageNoToken', value: 'Records per page', comment: 'Shows as a title list of values for page number (without the slot/token value)' },
+    RecordsSelected: { id: 'RecordsSelected', value: 'Record(s) Selected', comment: 'text describing a selected objects' },
```
Comment (tmcconechy) [inline src/components/locale/cultures/en-US.js:452]:
```suggestion
    RecordsSelected: { id: 'RecordsSelected', value: 'Record(s) Selected', comment: 'Text the number of items selected in a list component' },
```

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@tjamesallen15 see this comment
https://github.com/infor-design/enterprise/issues/6528#issuecomment-1438727974
We will need to check the count and show the correct label based on the count.

**Comment by tjamesallen15** [comment]
> @tjamesallen15 see this comment [#6528 (comment)](https://github.com/infor-design/enterprise/issues/6528#issuecomment-1438727974) We will need to check the count and show the correct label based on the count.

copy!

**Review by ericangeles — DISMISSED** [review]

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@tjamesallen15 i think we need to use 3 strings exactly as follows:

`0 Records Selected` -> (In English, "0 Records" is plural but in most languages, 0 is singular)
`1 Record Selected`
`{0} Records Selected` -> more than one

We can't just reuse the same one because in some languages 0, 1 or multiple "things" are translated different. So expect the logic to have three things and use the right one if its 0, 1, or more.

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**